app/models/service_template_ansible_playbook.rb

```diff
@@ -151,4 +151,17 @@ def update_catalog_item(options, auth_user = nil)
     end
     super
   end
+
+  def destroy(auth_user = 'system')
```

Comment:
If we can set an accessor for `auth_user` - but that sounds like a custom pattern.  Not sure how to allow for passing in any user - unless we store the user used to create the catalog_item in the `options` hash?  - Definitely open to ideas.